146: Bilinear Interpolation refinements and nodata management

dromagnoli opened the pull request.

aaime commented on jt-scale/src/test/java/it/geosolutions/jaiext/scale/BilinearScaleTest.java:
```diff
@@ -125,7 +123,46 @@ public void testImageScalingBinary() {
+                if (i > halfY && j > halfX) {
+                    assertTrue("Expected noData value but found different value", value == noDataValue);
+                } else {
+                    assertTrue("Expected valid value but found nodata", value > 0);
```
If nodata is 1, shouldn't this test say value > 1 ?

dromagnoli commented:
@aaime. You are right. I have changed the noDataValue before the commit but I forgot to update the check on the test, accordingly. I have updated it right now.
